validation/src/main/java/se/fortnox/reactivewizard/validation/ValidatorUtil.java

```diff
@@ -0,0 +1,52 @@
+package se.fortnox.reactivewizard.validation;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+import java.util.Set;
+
+import javax.inject.Inject;
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.executable.ExecutableValidator;
+
```

Comment:
Since the validator used is singleton this validator util could be singleton as well